### DISCUSSION TITLE
Sensor 2 strange behaviour

### DISCUSSION THREAD

**Discussion by Surreal62** [srozum/film_camera_tester#44] [Q&A]
Recently completed populating boards and programmed Nano with rev.2.9.1 firmware.
All buttons, rotary encoder working. Light unit 2 working perfectly.
Sensor 2 is found on Socket A, is activated and a test run starts, but there are no readings being displayed.
However, if wire 4 (Socket A, pos 4 pin 3 feeding D4 on Nano which has no connection to the sensor) insulation is touched, readings are made in a continuous, repeating cycle if the light sensor receives light. Cut off light from the sensor and the readings stop and there is a results display, so the photocell itself appears fine. Note: only when the wire A/B at the socket/shield side is touched is there any activity
I have checked all circuitry, wiring, component values and soldering, and cannot see any errors.
Any help would be greatly appreciated.
Thanks.

**Comment by srozum** [comment]
Duplicate of #40

**Comment by Surreal62** [comment]
Thank you!
I had actually read #40 and completely missed the connection!
I now have the wiring corrected to get the output of the sensor (A) to D4.
However, now I am getting no results at all. 
Best wishes,
Rob.
